bots/faq_bot.py

```diff
@@ -13,6 +13,9 @@
 SESSION_TIMEOUT_SECONDS = 300
 UNKNOWN_ANSWER = "Sorry, I can't help with that yet. Try to ask another question!"
 UNKNOWN_THRESHOLD = 0.5
+DEFAULT_WELCOME_MESSAGE = "Greetings! I'm the taiwan chat bot! You may ask me anything about taiwan and I'll do my best to answer your questions 🧙 For starters, you may select a question from below 👇"
```

Comment:
DEFAULT_WELCOME_MESSAGE = "Greetings! Taiwan chat bot! You may ask me anything about Taiwan and I'll do my best to answer your questions 🧙 For starters, you may select a question from below 👇"

Reply:
lol this sounds funny -> "Greetings! Taiwan chat bot!" The bot is calling his own name. I think i'll just drop "Taiwan chat bot" here.